colorbleed/plugins/maya/load/load_camera.py

```diff
@@ -27,8 +27,15 @@ def process_reference(self, context, name, namespace, data):
                           returnNewNodes=True)
 
         cameras = cmds.ls(nodes, type="camera")
-        for camera in cameras:
-            cmds.camera(camera, edit=True, lockTransform=True)
+
+        # Check the Maya version, lockTransform has been introduced since
+        # Maya 2017
```

Comment:
It's funny how you say it's introduced in Maya 2017 but then force the check to only allow Maya 2018 and up. haha. :)